core/testdata/directory_batch_get_user.json

```diff
@@ -1,7 +1,7 @@
 {
 	"directoryId": "integration",
 	"log": {
-		"treeId": "5360167840861150065",
+		"treeId": "7381771154556655958",
```

Comment:
It's not immediately clear to me why these tree ID updates are necessary, and why they are coupled to this CL?

Reply:
`treeId`, public keys, and various other bits in these test vectors are generated by reading from `/dev/random`.  This is less than ideal, but fixing it means patching out the randomness generators in Trillian which is a bit of a dangerous proposition.

Reply:
The reason these files are being *updated* is that the test vectors have changed due to the removal of "Previous"

Reply:
Since making Previous optional rather than removed, the test vectors don't need updating.